.github/prompts/conventions.md

```
@@ -0,0 +1,157 @@
+- `GitHub` · `GitLab` · `Kubernetes` · `JSON` · `YAML` · `URL` / `URLs` · `SSH`
+- `email`, not e-mail
+- `log in` / `sign in` as verbs; `login` as a noun only
+- Product and feature names are proper-cased when naming the product: Mattermost,
+  Mattermost Server, Mattermost Cloud, Channels, Playbooks, Boards, Copilot, Calls.
+  The common nouns are not: "public channels", "recent calls", "three boards".
+
+## Heading case
+
+Sentence case: capitalise the first word and proper nouns only.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)